sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -268,6 +268,14 @@ class DataFrameReader private[sql](sqlContext: SQLContext) {
   }
 
   /**
+   * Loads an ORC file and returns the result as a [[DataFrame]].
+   *
+   * @param path input path
+   * @since 1.5.0
+   */
+  def orc(path: String): DataFrame = format("orc").load(path)
```

Comment:
I did consider this but didn't add multiple-path support in this PR intentionally. The problem here is that currently we can't specify multiple paths via data source options, while DataFrame reader/writer API relies on the "path" option to find the input path. If you check `DataFrameReader.parquet`, you may see it's specially implemented, which I don't think is a good approach.

I'm thinking about adding multiple value support for data source options in a more general way. (Maybe just simple comma separated lists with proper comma escaping.)